src/helpers/genericTypeHelper.ts

```diff
@@ -0,0 +1,26 @@
+import { consts, texts } from '../config';
+import { GenericType } from '../types';
+
+export const getGenericItemDetailTitle = (genericType: GenericType): string => {
+  switch (genericType) {
+    case GenericType.Commercial:
+      return texts.commercial.commercial;
+    case GenericType.Job:
+      return texts.job.job;
+    default:
+      return '';
+  }
+};
+
+const { ROOT_ROUTE_NAMES } = consts;
+
+export const getGenericItemRootRouteName = (genericType: GenericType): string => {
+  switch (genericType) {
+    case GenericType.Commercial:
+      return ROOT_ROUTE_NAMES.COMMERCIALS;
+    case GenericType.Job:
+      return ROOT_ROUTE_NAMES.JOBS;
```

Comment:
this will grow with upcoming generic types, right?

Reply:
as does this